xmrstak/jconf.cpp

```diff
@@ -107,8 +107,9 @@ xmrstak::coin_selection coins[] = {
 	{ "cryptonight_v7_stellite", {POW(cryptonight_stellite)},  {POW(cryptonight_monero_v8)}, nullptr },
 	{ "cryptonight_gpu",         {POW(cryptonight_gpu)},       {POW(cryptonight_gpu)},       "pool.ryo-currency.com:3333" },
 	{ "cryptonight_conceal",     {POW(cryptonight_conceal)},   {POW(cryptonight_monero_v8)}, nullptr },
+	{ "cryptonight_v8_reversewaltz",{POW(cryptonight_v8_reversewaltz)}, {POW(cryptonight_monero_v8)}, "mining.grftpool.com:3333" },
 	{ "freehaven",               {POW(cryptonight_superfast)}, {POW(cryptonight_monero_v8)}, nullptr },
-	{ "graft",                   {POW(cryptonight_monero_v8)}, {POW(cryptonight_monero_v8)}, nullptr },
+	{ "graft",                   {POW(cryptonight_v8_reversewaltz)}, {POW(cryptonight_monero_v8)}, "mining.grftpool.com:3333" },
```

Comment:
Please change to `{POW(cryptonight_v8_reversewaltz), <forkblockversion>, POW(cryptonight_monero_v8)}, {POW(cryptonight_monero_v8)}` to allow to fork on a given block height. <forkblockversion> must be replaced with the block version where the coin is forking.

Please remove the pool sugestion or add an active pool. If you add an active pool please use not th elargest.

Reply:
- modified `"graft"` record as you asked;
- added fork block version - 12;
- removed pool suggestion. It is active since we maintain it, but it isn't the most popular Graft mining pool and, unfortunately, it has a zero hash rate.
Thanks.